src/org/javaday/java8/misused/stream/ExternalCollectionForGrouping.java

```diff
@@ -19,11 +19,9 @@
 
 import static java.util.stream.Collectors.*;
 
-/**
- * @author Alimenkou Mikalai
- * @version 1.0
- */
+//TODO: Decide whether to show or not as it's a bit complex example as for me
```

Comment:
I propose to show it at the end of the talk, but it is really interesting to avoid perception that talk is based on primitive cases only.